docs/reference/commandline/run.md

```diff
@@ -262,6 +262,13 @@ https://docs.docker.com/engine/installation/binaries/#/get-the-linux-binary)),
 you give the container the full access to create and manipulate the host's
 Docker daemon.
 
+On Windows, the paths must be specified using Windows-style semantics. Windows
+containers do not support the target of a volume or bind-mount being pre-
+populated in the container image.
```

Comment:
> Windows containers do not support the target of a volume or bind-mount being pre-populated in the container image.

Not sure I understand this part; Linux does not support this either? It's either

- `-v c:\target`, anonymous volume
- `-v namedvolume:c:\target`, named volume
- `-v c:\foo:c:\target`, bind-mount host directory (**not** a volume btw)